website/docs/components/icon.mdx

```diff
@@ -82,63 +82,26 @@ Here's a working example:
 
 ## Adding custom icons
 
-All Chakra icons are stored in the theme object under the `icons` key. You can
-extend this object to add your own icons. Here are the steps:
-
-- Export the icon's `svg` from Figma, Sketch, etc.
-- Use a tool like [SvgOmg](https://svgomg.firebaseapp.com) to reduce the size
-  and minify the markup.
-- Add the icon to the theme object.
-
-> Add the `fill=currentColor` attribute to the `path` or `g` so that when you
-> use `<Icon color="gray.200"/>`, it works correctly.
```

Comment:
Removed as icons are no longer part of theme. I did move the instructions (with fixed SvgOmg link) to another location.